torchvision/transforms/v2/_container.py

```diff
@@ -149,11 +158,19 @@ def __init__(
         self.transforms = transforms
         total = sum(p)
         self.p = [prob / total for prob in p]
+        self.max_transforms = max_transforms
 
     def forward(self, *inputs: Any) -> Any:
         idx = int(torch.multinomial(torch.tensor(self.p), 1))
         transform = self.transforms[idx]
-        return transform(*inputs)
+
+        if self.p:
+            return transform(*inputs)
+
+        else:
+            selected_transforms = random.sample(self.transforms, k=random.randint(1, self.max_transforms))
+            random.shuffle(selected_transforms)
+            return Compose(selected_transforms)(*inputs)
```

Comment:
Let's try to implement that without using the builtin `random` module and instead just rely on pytorch's RNG. `torch.randperm` can probably be used for that